magefile.publish.go

```diff
@@ -176,3 +177,97 @@ func PublishS3() error {
 
```

Comment:
What is purpose of leaving `PublishS3()` above (and related code)?

Reply:
Even if one will want, we likely will complain :)

Reply:
Removed